README.md

```diff
@@ -37,44 +15,56 @@ When using Maven, add the following dependency to your `pom.xml` file:
         </dependency>
 ```
 
+## Initialize the SDK
+
 Go to the settings page of your Castle account and find your **API Secret** and (optionally) your **APP ID**.
-Then create a new file named `castle_sdk.properties` with the following content:
 
-```properties
-app_id=<API_Secret>
-api_secret=<APP_ID>
+**Alt 1. Initialize using ENV variables**
 
-```
+On initialization the Castle SDK will look for the secret in `CASTLE_SDK_API_SECRET`.
```

Comment:
I would be more specific here: On initialization the Castle SDK will look for the secret in the `CASTLE_SDK_API_SECRET` environment variable.